src/components/SignUp.js

```diff
@@ -34,7 +34,8 @@ export class SignUp extends React.Component {
         this.setState({ error: "Invalid username and/or password" });
       }
     } catch (err) {
-      this.setState({ error: err.response.data });
```

Comment:
why did you make this change? we want to communicate the issue google gives back for why username or password is invalid